x-pack/plugins/integration_assistant/public/components/create_integration/create_integration_landing/create_integration_landing.tsx

```diff
@@ -6,150 +6,70 @@
  */
 
 import React from 'react';
-import {
-  EuiButton,
-  EuiCard,
-  EuiFlexGroup,
-  EuiFlexItem,
-  EuiIcon,
-  EuiLink,
-  EuiSpacer,
-  EuiText,
-  EuiTitle,
-  useEuiTheme,
-} from '@elastic/eui';
+import { EuiFlexGroup, EuiFlexItem, EuiIcon, EuiLink, EuiSpacer, EuiText } from '@elastic/eui';
 import { KibanaPageTemplate } from '@kbn/shared-ux-page-kibana-template';
-import { AssistantAvatar } from '@kbn/elastic-assistant';
-import { css } from '@emotion/react';
 import { FormattedMessage } from '@kbn/i18n-react';
-import { useAuthorization } from '../../../common/hooks/use_authorization';
-import {
-  AuthorizationWrapper,
-  MissingPrivilegesTooltip,
-} from '../../../common/components/authorization';
+import { AuthorizationWrapper } from '../../../common/components/authorization';
+import { AvailabilityWrapper } from '../../../common/components/availability_wrapper';
 import { IntegrationImageHeader } from '../../../common/components/integration_image_header';
 import { ButtonsFooter } from '../../../common/components/buttons_footer';
 import { SectionWrapper } from '../../../common/components/section_wrapper';
 import { useNavigate, Page } from '../../../common/hooks/use_navigate';
+import { IntegrationAssistantCard } from './integration_assistant_card';
 import * as i18n from './translations';
 
-const useAssistantCardCss = () => {
-  const { euiTheme } = useEuiTheme();
-  return css`
-    /* compensate for EuiCard children margin-block-start */
-    margin-block-start: calc(${euiTheme.size.s} * -2);
-  `;
-};
-
-const IntegrationAssistantCard = React.memo(() => {
```

Comment:
This component has been extracted to a separate file:
https://github.com/elastic/kibana/pull/187105/files#diff-8a3f4a5ae412a8b424468e3280a4f2f672a495a98225a345064469759dd86895